drivers/bluetooth/CMakeLists.txt

```
@@ -3,3 +3,7 @@
 zephyr_library()
 zephyr_library_include_directories(${ZEPHYR_BASE}/subsys/bluetooth)
 add_subdirectory(hci)
+
+if(${CONFIG_BT_NO_DRIVER})
```

@tejlmand FYI

---

it is not wrong, but the `${}` can be omitted.

And as this is generally what we do, I recommend changing to:
```suggestion
if(CONFIG_BT_NO_DRIVER)
```

---

Thanks @tejlmand. @theob-pro please do make the change.